<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="1" passed="0" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2023-11-12T18:47:02Z" name="Default suite" finished-at="2023-11-12T18:48:23Z" duration-ms="80965">
    <groups>
    </groups>
    <test started-at="2023-11-12T18:47:02Z" name="Default test" finished-at="2023-11-12T18:48:23Z" duration-ms="80965">
      <class name="com.basicautomation.BasicAutomation">
        <test-method is-config="true" signature="startBrowser()[pri:0, instance:com.basicautomation.BasicAutomation@345f69f3]" started-at="2023-11-13T00:46:58Z" name="startBrowser" finished-at="2023-11-13T00:47:02Z" duration-ms="4077" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- startBrowser -->
        <test-method signature="webAutomation()[pri:0, instance:com.basicautomation.BasicAutomation@345f69f3]" started-at="2023-11-13T00:47:02Z" name="webAutomation" finished-at="2023-11-13T00:48:23Z" duration-ms="80951" status="FAIL">
          <exception class="java.lang.UnsupportedOperationException">
            <message>
              <![CDATA[To submit an element, it must be nested inside a form element]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.UnsupportedOperationException: To submit an element, it must be nested inside a form element
	at org.openqa.selenium.remote.RemoteWebElement.submit(RemoteWebElement.java:86)
	at com.basicautomation.BasicAutomation.webAutomation(BasicAutomation.java:79)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.UnsupportedOperationException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- webAutomation -->
        <test-method is-config="true" signature="close()[pri:0, instance:com.basicautomation.BasicAutomation@345f69f3]" started-at="2023-11-13T00:48:23Z" name="close" finished-at="2023-11-13T00:48:24Z" duration-ms="1478" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- close -->
      </class> <!-- com.basicautomation.BasicAutomation -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
